math/minuit2/src/MnFunctionCross.cxx

```diff
@@ -46,7 +46,7 @@ MnCross MnFunctionCross::operator()(std::span<const unsigned int> par, std::span
    // for finding the point :
    double tlf = tlr * up;
    double tla = tlr;
-   unsigned int maxitr = 15;
+   unsigned int maxitr = 30;
```

Comment:
Just a first question here: why is increasing `maxitr` required here? Can this be explained in the commit description?

Reply:
Done!

Maximum iteration number in Minos algorithm is arbitrarily set (does not depend on particular instance).
We choose to increase it in order to recover Minos convergence for some test cases in `testMinimizer`.